src/components/WishList/CardContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from "@emotion/styled";
+
+export const WishListContainer = () => {
+  return (
+    <Cont>
+      <TitleCont>
+        <Title>위시리스트</Title>
+        <Quantity>3</Quantity>
+      </TitleCont>
+    </Cont>
```

Comment:
p2: 전 PR과 동일하게 네이밍만 봐서는 뭘 뜻하는지 전혀 알 수가 없습니다.  좀 더 명시적으로 구체적으로 적어주시면 좋을 것 같습니다.
```suggestion
    <Container>
      <TitleWrapper>
        <Title>위시리스트</Title>
        <Quantity>3</Quantity>
      </TitleWrapper>
    </Container>
```